Code/GraphMol/RGroupDecomposition/CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ rdkit_catch_test(rgroupCatchTests catch_rgd.cpp
 
 find_package(Boost ${RDK_BOOST_VERSION} COMPONENTS program_options)
 if(RDK_BUILD_CPP_TESTS AND Boost_FOUND)
-	add_executable(gaExample GaExample.cpp)
+	add_executable(gaExample GaExample.cpp RGroupDecompParams.h)
```

Comment:
Why is the header file in add executable?  Is this because It's missing in the header exports?

Reply:
I don't recall putting that there- perhaps it was CLion "helping".  I've removed it